new-log-viewer/src/services/MainWorker.ts

```diff
@@ -19,27 +35,46 @@ const postResp = <T extends WORKER_RESP_CODE>(
     postMessage({code, args});
 };
 
-onmessage = (ev: MessageEvent<MainWorkerReqMessage>) => {
+onmessage = async (ev: MessageEvent<MainWorkerReqMessage>) => {
     const {code, args} = ev.data;
     console.log(`[Renderer -> MainWorker] code=${code}: args=${JSON.stringify(args)}`);
 
-    switch (code) {
-        case WORKER_REQ_CODE.LOAD_FILE: {
-            postResp(WORKER_RESP_CODE.PAGE_DATA, {
-                logs: "Hello world!",
-                lines: new Map([
-                    // eslint-disable-next-line @stylistic/js/array-element-newline
-                    [1, 1],
-                ]),
-                startLogEventNum: 1,
+    try {
+        switch (code) {
+            case WORKER_REQ_CODE.LOAD_FILE: {
+                LOG_FILE_MANAGER = await LogFileManager.create(
+                    args.fileSrc,
+                    args.pageSize,
+                    args.decoderOptions
+                );
+
+                postResp(
+                    WORKER_RESP_CODE.NUM_EVENTS,
+                    {numEvents: LOG_FILE_MANAGER.numEvents}
```

Comment:
Our guidelines in other languages are not to allow public access of members in non-trivial classes (since they could be set without calling the relevant methods). Any reason to make this public?